src/test/java/teammates/test/cases/browsertests/InstructorEditStudentFeedbackPageUiTest.java

```diff
@@ -55,7 +76,7 @@ private void testEditResponse() throws Exception {
                                                   "student1InIESFPTCourse@gmail.tmt",
                                                   "student1InIESFPTCourse@gmail.tmt");
 
-        assertEquals("Student 1 self feedback.", fr.getResponseDetails().getAnswerString());
+        assertEquals("<p>Student 1 self feedback.</p>", fr.getResponseDetails().getAnswerString());
```

Comment:
It gave an assertion error on running test. Checking the other asserts in the file I don't know why this assert didn't have a `<p>/<p>` tag. So, I had to change to get green on tests